ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -741,7 +749,62 @@
           - email:
               notify-every-unstable-build: true
               recipients: projectantrea-dev@googlegroups.com
-          wrappers: []
+          wrappers:
+          - credentials-binding:
+            - text:
+                credential-id: WORKFORCE_POOL # Jenkins secret that stores client id
+                variable: WORKFORCE_POOL
+      - 'cloud-{name}-{test_name}':
+          test_name: credential-for-period
+          node: antrea-credential
+          description: 'This is the cloud credential job for antrea gke tests.'
+          builders:
+          - shell: |-
+              #!/bin/bash
+              set +x
+              token_file="/var/lib/jenkins/ci_properties.txt"
+              if [ -e "$token_file" ]; then
+                  modification_timestamp=$(stat -c %Y "$token_file")
+                  current_timestamp=$(date +%s)
+                  time_difference=$((current_timestamp - modification_timestamp))
+              else
+                  time_difference=36001
+              fi
+              # The credentials are valid for 12 hours, to allow sufficient time for job execution, the reuse threshold is set to 10 hours.
+              threshold=36000
+              if [ "$time_difference" -gt "$threshold" ]; then
+                  echo "Generate the new Cloud Credential"
+                  json_data=$(python3 get_access_using_api_client.py ${CLOUD_CLIENT_ID} ${CLOUD_CLIENT_TOKEN} PowerUser gcp ${GKE_RESOURCE_ID} 43200 prd)
+                  json_data=$(echo $json_data | sed "s/'/\"/g" | sed 's/True/true/')
+                  id_token=$(echo $json_data | jq -r '.credential.idToken')
+                  echo "ID_TOKEN=$id_token" > ${WORKSPACE}/ci_properties.txt
+                  cp ${WORKSPACE}/ci_properties.txt $token_file
+              else
+                  echo "Reuse the unexpired Cloud Credential"
+              cp $token_file ${WORKSPACE}/ci_properties.txt
```

Comment:
Indentation: it seems the the cp command needs a few spaces before it, so it can be aligned with the above echo.

Reply:
Updated, thanks.